src/pages/create-page/index.tsx

```diff
@@ -1,26 +1,25 @@
-import "./styles.css";
 import { Link } from "react-router-dom";
+import Input from "../../ui/input";
+import { Textarea } from "../../ui/textarea";
+import "./styles.css";
+import { Button } from "../../ui/button";
 
 const CreatePage = () => {
   return (
     <div className="container-notas">
       <h1 className="titulo-pagina">Crie uma nota</h1>
 
       <form>
-        <label htmlFor="titulo">Título</label>
-        <input type="text" id="titulo" />
-
-        <label htmlFor="descricao">Descrição</label>
-        <textarea name="descricao" id="descricao" maxLength={100}></textarea>
+        <Input labelText={"Título"} />
+        <Textarea labelText={"Descrição"} />
```

Comment:
`<Textarea labelText={"Descrição"} />` poderia deixar de usar `{}` quando for _string_